workflow-templates/staticwebapp-node-built-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+# For official documentation on this workflow, please refer to the link below.
+# https://github.com/brandwatch/bw-workflow-actions/blob/production/workflows/static-webapp/static-webapp-node-built-deploy.md
+
+name: CI Pipeline - Build Using Node & Deploy Static Webapp
+
+permissions:
+  checks: write
+  contents: read
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-pipeline:
+    name: "Build Pipeline"
+    uses: brandwatch/bw-workflow-actions/.github/workflows/static-webapp-node-built-deploy.yaml@production
+    with:
+      node-version: 18 # Current LTS versions are supported
```

Comment:
NABD but it might be prudent to add `test` and `node-build-script` so that people could theoretically have this working without actually needing to read the docs

Reply:
So node-build-script should be a convention thing. My hope is to push the devs to having a `build-website` node script defined.

Reply:
Test is a good idea though :-)